app/code/Magento/InventorySalesAdminUi/Ui/Component/Listing/Column/SalesChannels.php

```diff
@@ -70,10 +71,16 @@ private function prepareSalesChannelData(array $salesChannelData): array
         $preparedChannelData = [];
         foreach ($salesChannelData as $type => $salesChannel) {
             foreach ($salesChannel as $code) {
-                $preparedChannelData[$type][] = [
-                    'name' => $this->salesChannelNameResolver->resolve($type, $code),
-                    'code' => $code,
-                ];
+                try {
+                    $preparedChannelData[$type][] = [
+                        'name' => $this->salesChannelNameResolver->resolve($type, $code),
+                        'code' => $code,
+                    ];
+                } catch (NoSuchEntityException $e) {
+                    // We must handle the scenario in which a previous website was removed/renamed
```

Comment:
I'm not sure that is good idea to skip exception... especially if we fixed initial case

Reply:
I agree, but this is the only way to fix the already done mistake.